src/components/sections/time-travel/fundamentals/index.tsx

```diff
@@ -69,8 +69,8 @@ export const Fundamentals = () => {
       className={s.section}
       data-sitemap
       data-sitemap-icon="fundamentals"
-      data-sitemap-short-title="Fundamentals"
-      data-sitemap-complete-title="Fundamentals of time travel"
+      data-sitemap-short-title="Features"
```

Comment:
I wonder if we could do

* Fundamentals
* Features of time travel

My thinking is that these are not user facing features, so i want to be careful that we're talking about the tech itself